orchagent/main.cpp

```diff
@@ -149,6 +149,20 @@ int main(int argc, char **argv)
                 exit(EXIT_FAILURE);
             }
             break;
+        case 't':
+        {
+            gMacAgingTimeSet = true;
+            int tmptime = stoi(optarg);
+            if (tmptime < 0)
+            {   
+                macAgingTime = 0;
+                SWSS_LOG_DEBUG("Input macAgingTime < 0, set macAgingTime to 0");
```

Comment:
use SWSS_LOG_WARN instead

Reply:
We don't need to set the default value explicitly. SAI has set the default value in ocp_sai/include/sai/saiswitch.h :
/*
    * Dynamic FDB entry aging time in seconds [sai_uint32_t]
    *   Zero means aging is disabled.
    *  (default to zero)
*/